Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -111,9 +122,16 @@ public sealed class Server : ServerBase, IDisposable {
 
             return new InitializeResult {
                 capabilities = new ServerCapabilities {
-                    completionProvider = new CompletionOptions { resolveProvider = true },
                     textDocumentSync = new TextDocumentSyncOptions { openClose = true, change = TextDocumentSyncKind.Incremental },
-                    hoverProvider = true
+                    completionProvider = new CompletionOptions {
+                        triggerCharacters = new[] { "." },
+                        resolveProvider = true
+                    },
+                    hoverProvider = true,
+                    signatureHelpProvider = new SignatureHelpOptions { triggerCharacters = new[] { "(,)" } },
+                    // https://github.com/Microsoft/vscode-python/issues/1061
```

Comment:
Is there an issue in the PTVS repo for this?

Reply:
https://github.com/Microsoft/PTVS/issues/3964

Reply:
Link both from the comment, so we can resolve the correct issue when it gets fixed.